### ISSUE TITLE
Test ToolBoxController

### ISSUE THREAD

**Issue by RoeyMaor** [SpartanRefactoring/Main#1386]
<a href="https://github.com/AnnaBel7"><img src="https://avatars1.githubusercontent.com/u/22842727?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [AnnaBel7](https://github.com/AnnaBel7)**
_Sunday May 07, 2017 at 10:10 GMT_
_Originally opened as https://github.com/TechnionYP5777/Leonidas-FTW/issues/209_

----

**Comment by RoeyMaor** [comment]
<a href="https://github.com/amirsagiv83"><img src="https://avatars2.githubusercontent.com/u/15860047?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [amirsagiv83](https://github.com/amirsagiv83)**
_Friday May 19, 2017 at 10:10 GMT_

----

On hold until constructions are done.

**Comment by michalcohen** [comment]
Blocked by #1388

**Comment by amirsagiv83** [comment]
Unfortunately this was a nice to have issue but we don't have the time to test it.
Closing.
